webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtGce.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtGce = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('State')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.state}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Machine type')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.pretty_machine_type}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('zone')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.zone_name}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Created')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.creation_timestamp}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Private ip address')}</DescriptionListTerm>
```

Comment:
Please be consistent with the other entries because it makes life easier for translators.
```suggestion
      <DescriptionListTerm>{__('Private IP address')}</DescriptionListTerm>
```